File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
number = int(input())
count = 0
numbers = map(int, input().split())
for i in numbers:
    zero = 0
    if i == 1:
        continue
    for j in range(2, i):
        if i % j == 0:
            zero += 1
    if zero == 0:
        count += 1
print(count)

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int number = input.nextInt();

        int numbers;
        int count = 0;

        for (int i = 0; i < number; i++) {
            numbers = input.nextInt();
            for (int j = 2; j <= numbers; j++) {
                if (j == numbers){
                    count++;
                }
                if (numbers % j == 0){
                    break;
                }
            }
        }
        System.out.println(count);
    }
}